.github/workflows/camera_cq.yaml

```diff
@@ -12,8 +12,22 @@ jobs:
     - name: wemake-python-styleguide
       uses: wemake-services/wemake-python-styleguide@0.13.4
       with:
-        path: 'camera/'
+        path: 'camera/rpi-files/'
         reporter: 'github-pr-review'
       env:
         GITHUB_TOKEN: ${{ secrets.github_token }}
-
+  pytest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
you can think of migrating to pipenv instead of virtualenv like in django project part as ti si the newer solution and pipenv aims to combine Pipfile, pip and virtualenv into one command on the command-line (and now I think it is just going to be developed because as I ahve read there was long break in releases but now I see a few ones).

Reply:
Will do, I created #94 for this.